runtime/src/lib.rs

```diff
@@ -1247,11 +1247,11 @@ impl membership::Config for Runtime {
 
 parameter_types! {
     pub const MaxCategoryDepth: u64 = 6;
-    pub const MaxSubcategories: u64 = 40; // TODO: adjust
+    pub const MaxDirectSubcategoriesInCategory: u64 = 40;
+    pub const MaxTotalCategories: u64 = 40;
     pub const MaxThreadsInCategory: u64 = 20; // TODO: adjust
     pub const MaxPostsInThread: u64 = 20; // TODO: adjust
```

Comment:
No those values can be fixed based on https://gist.github.com/bedeho/1b231111596e25b215bc66f0bd0e7ccc: 
```suggestion
    pub const MaxDirectSubcategoriesInCategory: u64 = 5;
    pub const MaxTotalCategories: u64 = 40;
```